defaults/main.yml

```diff
@@ -50,9 +72,9 @@ atom_build_static_assets: "yes"
 
 # Themes (to build)
 atom_themes:
-  - path: "{{ atom_path }}/plugins/arDominionPlugin"
+  - path: "{{ atom_path }}/{{ atom_extra_path }}/plugins/arDominionPlugin"
```

Comment:
Why are you changing almost all the instances of atom_path to add atom_extra_path, instead of updating atom_path once and for all?

Reply:
Do you mean redefining `atom_path` with `set_fact`? 

There's a problem when using -e "@sites/whatever/site.yml" and the ansible variable precedence:

https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

The `atom_path` is order 21 in this case and we are trying to change with a task var (order 17).

I don't like using the new `atom_extra_path` variable either, but I haven't found an easier way to do it.

Reply:
A second alternative is to use a new variable (i.e `atom_src_path`) and define it in role with `set_fact`, but I think it is clearer using the `atom_extra_path` variable.

Reply:
I like @mamedin 's approach of using a separate variable `atom_extra_path` (I think redefining `atom_path` variable inside the role would be confusing and probably make debugging/troubleshooting harder)